server/src/main/scala/actors/DrtSystemInterface.scala

```diff
@@ -189,6 +188,12 @@ trait DrtSystemInterface extends UserRoleProviderLike with FeatureGuideProviderL
 
   lazy val manifestsProvider: (UtcDate, UtcDate) => Source[(UtcDate, VoyageManifests), NotUsed] = ManifestsProvider(manifestsRouterActor)
 
+  val testManifestsActor: ActorRef = system.actorOf(Props(new NoopActor), s"Noop-APIManifests")
+  val testArrivalActor: ActorRef = system.actorOf(Props(new NoopActor), s"Noop-LiveArrivals")
+  val testFeed: Feed[typed.ActorRef[Feed.FeedTick]] = Feed(TestFixtureFeed(system, testArrivalActor, Feed.actorRefSource), 1.second, 12.hours)
+
+  val restartActor: ActorRef = system.actorOf(Props(new NoopActor), "restart-actor-noop")
```

Comment:
The purpose of having these actors only in the `TestDrtSystemInterface` is to isolate them to make sure they're never available in the production app. We can't really move them into the main `DrtSystemInterface`